psi4/src/psi4/dfocc/separable_tpdm.cc

```diff
@@ -55,20 +55,20 @@ void DFOCC::separable_tpdm() {
         }
         */
 
-        // G_Q = \sum_{m,n} b_mn^Q G1c_mn
+        // G_Q = \sum_{m,n} b_mn^Q G1c_mn   eq. (41)
```

Comment:
@behnle What paper are these eqn. numbers from? I'm guessing DOI 10.1063/1.4896235?

Reply:
Almost. AFAIR it's Phys. Chem. Chem. Phys. (2016), 18, 11362, i.e. DOI: 10.1039/c6cp00164e

(also applies to the question above)

Reply:
I just checked, and the equation numbers from the two discussed papers do _not_ match the equation numbers in comments.

Reply:
looks like the below is what I have stashed. Shall I just remove the eqn numbers since they're not tracing cleanly and SB meant them as personal notes?

```
diff --git a/psi4/src/psi4/dfocc/lccd_pdm_3index_intr.cc b/psi4/src/psi4/dfocc/lccd_pdm_3index_intr.cc
index d05b6e26b..12901d747 100644
--- a/psi4/src/psi4/dfocc/lccd_pdm_3index_intr.cc
+++ b/psi4/src/psi4/dfocc/lccd_pdm_3index_intr.cc
@@ -57,14 +57,14 @@ void DFOCC::lccd_pdm_3index_intr() {
         U = std::make_shared<Tensor2d>("U2 (IA|JB)", naoccA, navirA, naoccA, navirA);
         ccsd_u2_amps(U, T1);
 
-        // T(Q,ia) = \sum_{jb} b_jb^Q U_ij^ab = \sum_{jb} b(Q,jb) U(jb,ia)  //CSB eq. (45) or (43), resp.
+        // T(Q,ia) = \sum_{jb} b_jb^Q U_ij^ab = \sum_{jb} b(Q,jb) U(jb,ia)  // https://doi.org/10.1039/C6CP00164E Eq. (45) or (43), resp.
         T = std::make_shared<Tensor2d>("T2 (Q|IA)", nQ, naoccA, navirA);
         T->gemm(false, false, bQiaA, U, 1.0, 0.0);
         T->write(psio_, PSIF_DFOCC_AMPS);
         T.reset();
 
         // Build G_mi and G_ae
-        // G_mi = \sum_{n,e,f} U_mn^ef L_in^ef = U(mn,ef) L(in,ef) //CSB correlation OPDM eq. (31) or (32), resp.
+        // G_mi = \sum_{n,e,f} U_mn^ef L_in^ef = U(mn,ef) L(in,ef)  // correlation OPDM https://doi.org/10.1039/C6CP00164E Eq. (31) or (32), resp.
         T = std::make_shared<Tensor2d>("U2 <IJ|AB>", naoccA, naoccA, navirA, navirA);
         T->sort(1324, U, 1.0, 0.0);
         U.reset();
@@ -72,13 +72,13 @@ void DFOCC::lccd_pdm_3index_intr() {
         L->sort(1324, T1, 1.0, 0.0);
         GijA->contract(false, true, naoccA, naoccA, naoccA * navirA * navirA, T, L, 1.0, 0.0);
 
-        // G_ae = -\sum_{m,n,f} U_mn^ef L_mn^af = L(mn,fa) U(mn,fe) //CSB correlation OPDM eq. (32) or (33), resp.
+        // G_ae = -\sum_{m,n,f} U_mn^ef L_mn^af = L(mn,fa) U(mn,fe)  // correlation OPDM https://doi.org/10.1039/C6CP00164E Eq. (32) or (33), resp.
         GabA->contract(true, false, navirA, navirA, naoccA * naoccA * navirA, L, T, -1.0, 0.0);
         T.reset();
         L.reset();
 
         // Build V_ijkl
-        // V_ijkl = \sum_{ef} T_ij^ef L_kl^ef //CSB eq. (50)
+        // V_ijkl = \sum_{ef} T_ij^ef L_kl^ef  // https://doi.org/10.1039/C6CP00164E Eq. (50)
         U = std::make_shared<Tensor2d>("T2 <IJ|AB>", naoccA, naoccA, navirA, navirA);
         U->sort(1324, T1, 1.0, 0.0);
         V = std::make_shared<Tensor2d>("V <IJ|KL>", naoccA, naoccA, naoccA, naoccA);
diff --git a/psi4/src/psi4/fnocc/df_ccsd.cc b/psi4/src/psi4/fnocc/df_ccsd.cc
```

Reply:
You may remove them entirely. I can also double-check them. I indeed mixed up some, so (41) should be (40).
Actually, in the case where OLCCD and OMP2 are identical, the equation numbers also can refer to 10.1021/ct500231c
(J. Chem. Theory Comput. 2014, 10, 2371−2378). So e.g. in the case of the OPDMs, these are (31)&(32) in the PCCP paper but (32)&(33) in the JCTC paper.
I admit that this is quite some mess and should be cleaned up.